x-pack/plugins/observability_shared/public/hooks/use_editable_settings.tsx

```diff
@@ -5,14 +5,20 @@
  * 2.0.
  */
 import { useKibana } from '@kbn/kibana-react-plugin/public';
-import { useMemo, useState } from 'react';
-import { FieldState } from '@kbn/advanced-settings-plugin/public';
-import { toEditableConfig } from '@kbn/advanced-settings-plugin/public';
+import React, { useMemo, useState } from 'react';
 import { IUiSettingsClient } from '@kbn/core/public';
 import { isEmpty } from 'lodash';
+import { getFieldDefinition } from '@kbn/management-settings-field-definition';
+import {
+  FieldDefinition,
+  OnFieldChangeFn,
+  UiSettingMetadata,
+  UnsavedFieldChange,
+} from '@kbn/management-settings-types';
+import { normalizeSettings } from '@kbn/management-settings-utilities';
 import { ObservabilityApp } from '../../typings/common';
 
-function getEditableConfig({
+function getFields({
```

Comment:
One small concern, `getFields` might be too generic, maybe something like `getSettingsFields`? Also might be worth renaming the file to match the updated hook name.